src/Grpc.Net.ClientFactory/Internal/GrpcHttpClientFactory.cs

```diff
@@ -120,4 +116,197 @@ public ObjectFactory Activator
             }
         }
     }
+
+
+    /// <summary>
```

Comment:
Pending removal. Had to make this private to ensure no type collisions in tests where a reference to Grpc.Core need to be added.